tests/scripts/run_pre_post_commit_regressions_multi_device.sh

```diff
@@ -41,9 +41,7 @@ pytest models/demos/falcon40b/tests/test_falcon_end_to_end.py::test_FalconCausal
 
 # Falcon40B 8 chip prefill tests; we need 8x8 grid size for prefill tests on wormhole_b0
 if [ "$ARCH_NAME" == "wormhole_b0" ]; then
```

Comment:
Also please get rid of this ARCH_NAME check - it's not necessary because of `skip_for_grayskull` and we actually no longer want to use it going forward.